File: ClientApp/ClientApp/Models/Responses/SignupResponse.cs
using ClientApp.Models.ResponseErrors;
using Newtonsoft.Json;

namespace ClientApp.Models.Responses
{
    class SignupResponse : Response
    {
        private SignupResponse() { }
        [JsonProperty]
        public SignupErrors Errors { get; private set; }
        public ResponsesCodes GetCode() => ResponsesCodes.SignupResponse;
    }
}

File: M_Trivia/M_Trivia/SignupResponse.cpp
#include "SignupResponse.hpp"
#include "json.hpp"


SignupResponse::SignupResponse(SignupResponseErrors* signupResponseErrors)
     : Response(signupResponseErrors){
}

ResponseCodes SignupResponse::getCode() const 
{
	return ResponseCodes::C_SignupResponse; 
}
